### PR TITLE
Fixes #7696: Missing reporting changes after #7644

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/7696

### CODE REVIEW

**Comment by amousset** [comment]
WIP, still untested.

**Comment by amousset** [comment]
Ready for review.

File: techniques/system/common/1.0/update.st [hunk 1]
```diff
@@ -316,11 +316,11 @@ bundle agent update_reports
...
     # There must be NO components in repair or error
-    rudder_ncf_common_updated_ok.rudder_ncf_local_updated_ok.(rudder_tools_updated_kept|(rudder_tools_updated_repaired.rudder_tools_updated_ok)).(root_server|rudder_promises_generated_ok|(rudder_promises_generated_repaired.config_ok)).!(rudder_promises_generated_repaired|rudder_promises_generated_error|rudder_tools_updated_error|rudder_tools_updated|rudder_tools_update_error|rudder_ncf_common_updated|rudder_ncf_common_update_error|rudder_ncf_local_updated|rudder_ncf_local_update_error|config|no_update)::
+    rudder_ncf_common_updated_ok.rudder_ncf_local_updated_ok.(policy_server|rudder_tools_updated_kept|(rudder_tools_updated_repaired.rudder_tools_updated_ok)).(root_server|rudder_promises_generated_ok|(rudder_promises_generated_repaired.config_ok)).!(rudder_promises_generated_repaired|rudder_promises_generated_error|rudder_tools_updated_error|rudder_tools_updated|rudder_tools_update_error|rudder_ncf_common_updated|rudder_ncf_common_update_error|rudder_ncf_local_updated|rudder_ncf_local_update_error|config|no_update)::
```
Comment (ncharles) [inline techniques/system/common/1.0/update.st]: this is weird, having the condition "policy_server or something" in the middle seems to be not what we expect

Comment (ncharles) [inline techniques/system/common/1.0/update.st]: ok, it makes senses after a lot of head scratching

**Pull request merged**